### DISCUSSION TITLE
Doc comment on Introduction to Weaviate (/developers/academy/zero_to_mvp/hello_weaviate/intro_weaviate)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#950] [Docs/blog comments]
# Doc comment on Introduction to Weaviate (/developers/academy/zero_to_mvp/hello_weaviate/intro_weaviate)

&nbsp;&nbsp;What is Weaviate?

https://weaviate.io/developers/academy/zero_to_mvp/hello_weaviate/intro_weaviate

**Comment by djjeffr** [comment]
Weaviate should adopt the abbreviation wv8.
